app/models/note.rb

```diff
@@ -12,6 +13,9 @@ class Note
 
   # Validations
   validates :created_by, :full_text, presence: true
+<<<<<<< HEAD
```

Comment:
Same deal -- go ahead and remove lines 16-27 here, since we're replacing with `Auditable`.